.github/scripts/update-release.js

```diff
@@ -1,21 +1,26 @@
 const core = require('@actions/core');
-const { GitHub } = require('@actions/github');
+const { Octokit } = require("@octokit/core");
 
 /**
  * Functionality from tubone24/update_release.
  * Link: https://github.com/tubone24/update_release
  */
 const updateRelease = async () => {
   try {
-    const github = new GitHub(process.env.REPO_ACCESS_TOKEN);
+    const octokit = new Octokit({
+      auth: process.env.REPO_ACCESS_TOKEN
+    })
     const [owner, repo] = process.env.REPO_NAME ? process.env.REPO_NAME.split('/') : [null, null];
     const tag = process.env.TAG_NAME;
-    const getReleaseResponse = await github.repos.getReleaseByTag({
+
+    //https://docs.github.com/en/rest/releases/releases#get-a-release-by-tag-name
+    const getReleaseResponse = await octokit.request('GET /repos/{owner}/{repo}/releases/tags/{tag}',{
```

Comment:
could we use the helper method mentioned here https://github.com/octokit/plugin-rest-endpoint-methods.js/blob/master/docs/repos/getReleaseByTag.md instead of calling the api directly.